pages/conference/_eventType/_id.vue

```diff
@@ -235,22 +233,13 @@ export default {
                 dateTag: this.getDateTag(beginTime),
             }
         },
-        getDatetimeFromString(datetimeString) {
-            const datetimeUtc = new Date(datetimeString)
-            const offset = new Date().getTimezoneOffset()
-            const datetime = new Date(datetimeUtc - offset)
-            return datetime
-        },
         getDateTag(beginTime) {
-            const month = beginTime.getMonth() + 1
-            const date = beginTime.getDate()
-            if (month === 10 && date === 2) {
+            const dayOneMidnight = new Date('2021-10-02 16:00:00')
```

Comment:
I'm curious why `dayOneMidnight` is this time?
Could we use ISO 8601 instead? To me, that will be more precise.

Reply:
This is the UTC time of the midnight in Taiwan (UTC+8).